<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsLayoutPictureWidgetBase</class>
 <widget class="QWidget" name="QgsLayoutPictureWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Picture Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 2px; font-weight: bold; background-color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string>Picture</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-4</y>
        <width>520</width>
        <height>881</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout">
       <item>
        <widget class="QRadioButton" name="mRadioSVG">
         <property name="text">
          <string>SVG image</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mRadioRaster">
         <property name="text">
          <string>Raster image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="mRasterFrame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QgsImageSourceLineEdit" name="mImageSourceLineEdit" native="true">
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="mRasterDDBtnFrame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QgsPropertyOverrideButton" name="mSourceDDBtn">
               <property name="text">
                <string>…</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="mSVGFrame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,0">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <widget class="QgsSvgSourceLineEdit" name="mSvgSourceLineEdit" native="true">
            <property name="focusPolicy">
             <enum>Qt::StrongFocus</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QFrame" name="mSvgDDBtnFrame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>SVG Groups</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTreeView" name="viewGroups">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>3</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="layoutWidget1">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>SVG Image</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListView" name="viewImages">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>5</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>250</height>
                 </size>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="iconSize">
                 <size>
                  <width>32</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="movement">
                 <enum>QListView::Static</enum>
                </property>
                <property name="resizeMode">
                 <enum>QListView::Adjust</enum>
                </property>
                <property name="layoutMode">
                 <enum>QListView::Batched</enum>
                </property>
                <property name="spacing">
                 <number>2</number>
                </property>
                <property name="gridSize">
                 <size>
                  <width>36</width>
                  <height>36</height>
                 </size>
                </property>
                <property name="viewMode">
                 <enum>QListView::IconMode</enum>
                </property>
                <property name="uniformItemSizes">
                 <bool>true</bool>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="mSVGParamsGroupBox">
         <property name="title">
          <string>SVG Parameters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QgsDoubleSpinBox" name="mStrokeWidthSpinBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="suffix">
             <string> mm</string>
            </property>
            <property name="decimals">
             <number>2</number>
            </property>
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.200000000000000</double>
            </property>
            <property name="value">
             <double>0.200000000000000</double>
            </property>
            <property name="showClearButton" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QgsPropertyOverrideButton" name="mFillColorDDBtn">
            <property name="text">
             <string>…</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="mStrokeColorLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Stroke color</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QgsColorButton" name="mFillColorButton">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="mStrokeWidthLabel">
            <property name="text">
             <string>Stroke width</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QgsColorButton" name="mStrokeColorButton">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Fill color</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QgsPropertyOverrideButton" name="mStrokeColorDDBtn">
            <property name="text">
             <string>…</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QgsPropertyOverrideButton" name="mStrokeWidthDDBtn">
            <property name="text">
             <string>…</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="mPreviewGroupBox">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="title">
          <string>Size and Placement</string>
         </property>
         <property name="syncGroup" stdset="0">
          <string notr="true">composeritem</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QComboBox" name="mResizeModeComboBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Placement</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Resize mode</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QComboBox" name="mAnchorPointComboBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QgsCollapsibleGroupBoxBasic" name="mRotationGroupBox">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="title">
          <string>Image Rotation</string>
         </property>
         <property name="syncGroup" stdset="0">
          <string notr="true">composeritem</string>
         </property>
         <property name="collapsed" stdset="0">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1">
          <item row="2" column="1">
           <widget class="QComboBox" name="mNorthTypeComboBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>North alignment</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="mRotationFromComposerMapCheckBox">
            <property name="text">
             <string>Sync with map</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QgsLayoutItemComboBox" name="mComposerMapComboBox"/>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QgsDoubleSpinBox" name="mPictureRotationSpinBox">
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string> °</string>
            </property>
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Offset</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QgsDoubleSpinBox" name="mPictureRotationOffsetSpinBox">
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string> °</string>
            </property>
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qgsscrollarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBoxBasic</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsPropertyOverrideButton</class>
   <extends>QToolButton</extends>
   <header>qgspropertyoverridebutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsLayoutItemComboBox</class>
   <extends>QComboBox</extends>
   <header>qgslayoutitemcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsImageSourceLineEdit</class>
   <extends>QWidget</extends>
   <header>qgsfilecontentsourcelineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsSvgSourceLineEdit</class>
   <extends>QWidget</extends>
   <header>qgsfilecontentsourcelineedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>mRadioSVG</tabstop>
  <tabstop>mRadioRaster</tabstop>
  <tabstop>mImageSourceLineEdit</tabstop>
  <tabstop>viewGroups</tabstop>
  <tabstop>viewImages</tabstop>
  <tabstop>mSvgSourceLineEdit</tabstop>
  <tabstop>mFillColorButton</tabstop>
  <tabstop>mFillColorDDBtn</tabstop>
  <tabstop>mStrokeColorButton</tabstop>
  <tabstop>mStrokeColorDDBtn</tabstop>
  <tabstop>mStrokeWidthSpinBox</tabstop>
  <tabstop>mStrokeWidthDDBtn</tabstop>
  <tabstop>mPreviewGroupBox</tabstop>
  <tabstop>mResizeModeComboBox</tabstop>
  <tabstop>mAnchorPointComboBox</tabstop>
  <tabstop>mRotationGroupBox</tabstop>
  <tabstop>mPictureRotationSpinBox</tabstop>
  <tabstop>mRotationFromComposerMapCheckBox</tabstop>
  <tabstop>mComposerMapComboBox</tabstop>
  <tabstop>mNorthTypeComboBox</tabstop>
  <tabstop>mPictureRotationOffsetSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
